<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>296</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>车辆管理</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>296</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="Car">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QTextEdit" name="textEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolBox" name="toolBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>379</width>
             <height>407</height>
            </rect>
           </property>
           <attribute name="label">
            <string>出售车辆</string>
           </attribute>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>50</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>厂家：</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>50</y>
              <width>87</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox_2">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>110</y>
              <width>87</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>110</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>品牌：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>180</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>报价：</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>170</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>180</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>万</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>230</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>数量：</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>230</y>
              <width>42</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>230</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>剩余数量：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>230</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>20</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>290</y>
              <width>68</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>数量：</string>
            </property>
           </widget>
           <widget class="QToolBox" name="toolBox_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>-20</y>
              <width>379</width>
              <height>453</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>379</width>
               <height>395</height>
              </rect>
             </property>
             <attribute name="label">
              <string>出售车辆</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>厂家：</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="factorycomboBox"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>品牌：</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="brandcomboBox"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>报价：</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pricelineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>万</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>数量：</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="numspinBox"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>剩余数量：</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lastlabel">
                  <property name="text">
                   <string>20</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>金额：</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="totallineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>万</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QPushButton" name="OKpushButton">
                  <property name="text">
                   <string>确定</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="cancelpushButton">
                  <property name="text">
                   <string>取消</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>379</width>
               <height>395</height>
              </rect>
             </property>
             <attribute name="label">
              <string>新车入库</string>
             </attribute>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>379</width>
             <height>407</height>
            </rect>
           </property>
           <attribute name="label">
            <string>新车入库</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Calc"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="actionCar"/>
    <addaction name="actionCalc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>系统设置</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCar">
   <property name="text">
    <string>车辆管理</string>
   </property>
  </action>
  <action name="actionCalc">
   <property name="text">
    <string>销售统计</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
